branca/utilities.py

```diff
@@ -387,19 +388,20 @@ def _camelify(out):
 
 
 def _parse_size(value):
-    try:
-        if isinstance(value, int) or isinstance(value, float):
-            value_type = "px"
-            value = float(value)
-            assert value > 0
+    if isinstance(value, (int, float)):
+        return float(value), "px"
+    elif isinstance(value, str):
+        match = re.fullmatch(r"([\d.]+)\s?(\w{1,5}|%)", value.strip())
```

Comment:
I'm not a regex expert but testing it with online tools seems to work as expect. The firs t is supposed to be the number and the second one the `%`, right?

Reply:
that's right! I added a comment to clarify